pkg/envoy/lds/listener.go

```diff
@@ -58,6 +58,8 @@ func (lb *listenerBuilder) newOutboundListener() (*xds_listener.Listener, error)
 	}
 
 	if featureflags := lb.cfg.GetFeatureFlags(); featureflags.EnableEgressPolicy {
+		listener.AccessLog = envoy.GetAccessLog()
```

Comment:
Please enable this by default on the listener and not just when Egress policy is enabled. This will be required for global passthrough egress and also for in-mesh TCP traffic.

Reply:
I'm curious if there is any specific reason why the log wasn't added for TCP listener initially? If it was ignored, then sure we can add it now.

Reply:
Updated. Added log for both inbound and outbound TCP listeners

Reply:
Thanks. Could you please update the PR && commit description and title to reflect the changes made.